tests/session/session-startTransaction_error-004.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() with wrong argument for options array on PHP 7.0
```

Comment:
This test on PHP 7.0 was necessary because of build failures: https://evergreen.mongodb.com/task_log_raw/mongo_php_driver_tests_php7__versions~4.2_php_versions~7.0_os_php7~debian92_test_test_replicaset_auth_a85e0922c9d1c2a82fb02eef6625d75b02721a79_19_08_29_14_45_15/0?type=T#L1425

I suspect that the issue stems from nullable types being introduced in PHP 7.1 only, which is why the error message omits "or null" on PHP 7.0.